config/output_files.yaml

```diff
@@ -23,3 +23,7 @@ files:
   - name: Pindel VCF file
     input: cnv_sv/pindel_vcf/{sample}_{type}.no_contig.vcf.gz
     output: vcf/{sample}_{type}.pindel.vcf.gz
+
+  - name: cnvkit vcf file
+    input: cnv_sv/cnvkit_vcf/{sample}_{type}.pathology.vcf
```

Comment:
Will anyone have TC from a pathology or should we just have something like pureCN? At least uppsala never has any TC. But i guess we can start with a "pathology" value until purceCN kinks have been worked out.

Reply:
Yeah, these are simply the values that get entered into the sample file, and this is a required column at the moment. I realise that this is not applicable in the same way as for solid tumors so something should definitely be done about this. I'll add it as a topic for the hackathon.